packages/experimental/RadioGroup/src/Radio/Radio.types.ts

```diff
@@ -82,7 +108,9 @@ export interface RadioSlotProps {
   root: IViewProps;
   button: IViewProps;
   innerCircle: IViewProps;
+  content: IViewProps;
```

Comment:
Can we get more specific with the `content` name? Looking at this, I'd assume this slot wraps the whole component. But looking at Radio.tsx, it looks like it's specific for the label + subtext.

Maybe we rename it `labelContent`?

Reply:
It saves the developer the extra step of having to figure out what content this slot is referring to

Reply:
Thanks for the suggestion, updated!